src/GameLoop.cpp

```diff
@@ -291,9 +291,13 @@ void GameLoop::UpdateAllButDraw(bool bRunningFromVBLANK)
 	* acting on song beat from last frame */
 	HandleInputEvents(fDeltaTime);
 
+	/* note this might be negatively impacting performance but we dont know 100% for sure. we should test this commented and uncommented.
+	outfox doesn't use it right now. outfox also doesn't have the LIGHTSMAN line below it at all, so i've commented it out too.
+	
 	//bandaid for low max audio sample counter
 	SOUNDMAN->low_sample_count_workaround();
 	LIGHTSMAN->Update(fDeltaTime);
```

Comment:
do not comment out the lights update, as this will stick the lights for the player, move this one outside of the comment :)